src/view/use-lift-instruction/use-lift-instruction.js

```diff
@@ -24,8 +23,8 @@ export default function useLiftInstruction(
       // add the description text
       el.textContent = liftInstruction;
 
-      // hide the element visually
-      Object.assign(el.style, visuallyHidden);
+      // prevent AT from idly reading it. Fixes #1568
+      Object.assign(el.style, { display: 'none' });
```

Comment:
Can you confirm that the lift instructions still work as expected?

Reply:
Works fine in VO (90% sure it also works in NVDA/JAWS too, but didn't test)